riotfile.py

```diff
@@ -1874,43 +1874,28 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "aiopg": "~=0.16.0",
                 "pytest-randomly": latest,
             },
-            # venvs=[
-            # FIXME: tests fail on aiopg 1.x
-            # Venv(
-            #     # aiopg dropped support for Python 3.5 in 1.1
```

Comment:
Assuming we no longer care about these very old versions of python

Reply:
That's right. 3.7 is the oldest that ddtracepy supports at present.